core/src/test/java/org/kohsuke/stapler/export/XMLDataWriterTest.java

```diff
@@ -55,7 +56,7 @@ public void testInheritance() throws Exception {
         @Exported @Override public String generic() {return "sub2";}
     }
     public void testInheritance2() throws Exception { // JENKINS-13336
-        assertEquals("<sub2><basic>super</basic><generic>sub2</generic></sub2>",
+        assertEquals("<sub2 _class='Sub2'><basic>super</basic><generic>sub2</generic></sub2>",
```

Comment:
Emitting the lowercased class name as the XML root element name now becomes superfluous, but I guess we cannot compatibly stop and pick a generic root element name like `<root>`.